files/build_templates/dhcp_relay.service.j2

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=DHCP relay container
-Requires=docker.service teamd.service
-After=swss.service teamd.service
+Requires=updategraph.service swss.service teamd.service
+After=updategraph.service swss.service teamd.service
```

Comment:
why dhcp_relay requires teamd service?

Reply:
Because it needs to listen on port channels. Please note that this was the original behavior and I didn't change it in this PR.